setup.py

```diff
@@ -49,22 +50,20 @@ def _hash(filepath):
         file_hash = base64.b64encode(hashlib.sha512(file_as_bytes).digest())
         return "sha512-" + file_hash.decode()
 
-    os.makedirs(external_dir_single, exist_ok=True)
+    external_dir_single.mkdir(exist_ok=True)
     # Used for downloading e.g. Leaflet single file
     if out_name is None:
         out_name = url.split("/")[-1]
-    dst_path = os.path.join(external_dir_single, out_name)
-    if os.path.isfile(dst_path):
-        if integrity_hash and (_hash(dst_path) == integrity_hash):
```

Comment:
Thanks for pointing this out! This looks like a bug introduced via https://github.com/projectmesa/mesa-geo/pull/61. It was meant to compare the hash with existing file if there's any, and avoids downloading if the hashes match.

Instead of removing it, I would suggest to fix the issue. Perhaps we could delete the `return` after `else` statement, and indent some of the code below into `else`?